Userland/Libraries/LibWeb/HTML/HTMLPreElement.cpp

```diff
@@ -36,4 +37,20 @@ void HTMLPreElement::apply_presentational_hints(CSS::StyleProperties& style) con
     });
 }
 
+// https://html.spec.whatwg.org/multipage/obsolete.html#dom-pre-width
+long HTMLPreElement::width() const
```

Comment:
We have WebIDL::Long for this now :^)

Reply:
You right those are great, I have updated the my code. And also took the liberty to update all the other instances of `long (long)` and `unsigned long (long)` in sixth commit.